react/tic-tac-toe/src/app/screens/Login/components/fields/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const getValidityClassName = meta => {
+  if (meta.active) {
+    return '';
+  }
+  if (meta.touched && meta.invalid) {
+    return " invalid";
+  }
+  if (meta.touched && meta.valid) {
+    return " valid";
+  }
+  return '';
+};
+
+
+export const customInput = props => {
```

Comment:
Unless you are planing to add different kind of inputs, i'd change some things.
First, this should be a component
```jsx
function CustomInput({ label, input, type, meta }) { ...
...
export default CustomInput;
```
(remember you can destructure the params)
Then, i'd change the directory structure. Probably `.../Login/components/CustomInput`